hack/test

```diff
@@ -60,8 +63,18 @@ if [ "$TEST_COVERAGE" = "1" ]; then
   coverageFlags="-coverprofile=/coverage/coverage.txt -covermode=atomic"
 fi
 
+buildxCmdArgs="--build-arg 'BUILDKITD_TAGS=$BUILDKITD_TAGS'"
+if [ ! -z "$HTTP_PROXY" ]; then
+    buildxCmdArgs="$buildxCmdArgs --build-arg 'HTTP_PROXY=$HTTP_PROXY'"
```

Comment:
You can just set `--build-arg HTTP_PROXY --build-arg HTTPS_PROXY` without the conditions.